gp-ecommerce-fields/gpecf-remove-field-label-from-option-label.php

```diff
@@ -3,6 +3,8 @@
  * Gravity Perks // eCommerce Fields // Remove Field Label from Option Label in Order Summary
```

Comment:
This reads strangely. Let's change it to:

`* Gravity Perks // eCommerce Fields // Remove Field Label from Option Fields in Order Summary`